apps/backend/app.py

```diff
@@ -185,12 +201,37 @@ def upload_experiment_results(expId, trialExtraFile, postProcess):
     uploadBlob = firebaseBucket.blob(f"results/result{expId}.csv")
     uploadBlob.upload_from_filename('results.csv')
 
+        # Upload to MongoDB
+
+    try:
+            mongoClient.admin.command('ping')
+    except ConnectionFailure:
+        print("Server not available (this is an error)")
+        
+    
+    print('Uploading to MongoDB')
+    experimentFile = open(f"results.csv") # there is probably a better way to do this
+    experimentData = experimentFile.read()
+    experimentFile.close()
+    experimentResultEntry = {"_id": expId,
+                        "resultContent": experimentData}
+    
+    resultId = mongoResultsCollection.insert_one(experimentResultEntry).inserted_id
+    print(f"inserted result csv into mongodb with id: {resultId}")
     if trialExtraFile != '' or postProcess:
         print('Uploading Result Csvs')
         try:
             shutil.make_archive('ResultCsvs', 'zip', 'ResCsvs')
             uploadBlob = firebaseBucket.blob(f"results/result{expId}.zip")
             uploadBlob.upload_from_filename('ResultCsvs.zip')
+            
+            # mongoResultsZipCollections
+            with open("ResultCsvs.zip", "rb") as file:
+                encoded = Binary(file.read())
+            experimentZipEntry = {"_id": expId,
+                            "fileContent": encoded}
+            resultZipId = mongoResultsZipCollections.insert_one(experimentZipEntry).inserted_id
```

Comment:
Ditto with the last comment about insert_one

Reply:
ditto